backend/pyproject.toml

```diff
@@ -85,7 +85,9 @@ module = [
     "socketio.*",
     "uptrace",
     "syncer",
-    "vertexai.language_models"
+    "vertexai.language_models",
```

Comment:
I would not add those as dependencies. We should never import the llm providers at top level of the file but always in methods. Then if a user wants to use it he/she will have to manually install this package.

The reason is that we don't want to bloat the package with too many dependencies.

Reply:
It's for mypy missing import error only.

Reply:
Then it should probably go with the rest of the [optional dependencies](https://github.com/Chainlit/chainlit/blob/main/backend/pyproject.toml#L72C1-L73C1)?